### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cometkim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,85 +58,38 @@ The version listed is used in current environment. This must not be identical to
...
     Error: Cannot find module 'generate'
     ```
-    - when you execute *local test* via `prisma deploy` command with `prisma generate` at **post-deploy hook setting** in `prisma.yml`, this error occured in prisma *Homebrew installation*.
+    - when you execute *local test* via `prisma deploy` command at **post-deploy hook setting** in `prisma.yml`, this error occured in prisma *Homebrew installation*.
     - that error not fixed in *Homebrew installation* yet.
     - **so install via npm**
     - [GitHub issues: Error: Cannot find module 'generate'](https://github.com/prisma/Mongo-Connector-Preview/issues/1#issuecomment-441361867)
    
 ### Running server
-1. `$ docker-compose up -d`
```
Comment (hyochan) [inline README.md]: @geoseong Your updates has been deleted by this `PR`. Please confirm!

Comment (cometkim) [reply README.md]: because we've dropped the prisma's API binding (for now)

File: README.md [hunk 2]
```diff
@@ -58,85 +58,38 @@ The version listed is used in current environment. This must not be identical to
...
+| PRISMA_DB_USER               | Database username         | true      | prisma                |
+| PRISMA_DB_PASSWORD           | Database password         | true      | prisma                |
+
+Then,
+
+1. `docker-compose up -d`  
+   Run all prisma services in background.
+
+2. `npm run prisma:deploy`  
+   Initialize or apply changes of prisma data modeal to database.
```
Comment (geoseong) [inline README.md]: @cometkim `prisma:deploy` script is not in `package.json` now. do you have any idea to use this script in the future? 
and the word`modeal` in **Initialize or apply changes of prisma data modeal to database.** maybe a typo.
This content looks much better except what I pointed out!

File: README.md (hunk 1 above)
Comment (geoseong) [reply README.md]: @hyochan @cometkim okay. no problem

File: README.md (hunk 2 above)
Comment (cometkim) [reply README.md]: Thanks @geoseong , I just fixed typo and script.

just `prisma deploy` looks good to me, we already have a guide to install prisma command in above section

Comment (geoseong) [reply README.md]: @cometkim ok I got it. I checked your modified content and looks perfect.

**Comment by cometkim** [comment]
Please do not merge yet

**Pull request merged**